lib/common/json-schema-validator.js

```diff
@@ -40,6 +42,9 @@ function jsonSchemaValidatorFactory(
      * @return {Boolean} validation pass or throw error
      */
     JsonSchemaValidator.prototype.validate = function (schema, data) {
+
+        sanitizer.decrypt(data);
```

Comment:
@yyscamper Hey Felix, so I took your feedback and changed my other PR (on-tasks). During my testing, I had reverted on-core to master and was able to pass the JSON validation. So it would seem the bug resided in on-tasks, so I will close this PR (on-core)... since based on my testing, it seems to be not needed.